hack/e2e-kind-cluster-setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -xe
+
+CNI_VERSION=${CNI_VERSION:-0.4.0}
+OCI_BIN=${OCI_BIN:-docker}
+IMG_REGISTRY=${IMAGE_REGISTRY:-localhost:5000/maiqueb}
+
+setup_cluster() {
+    git clone https://github.com/k8snetworkplumbingwg/multus-cni/
+    pushd multus-cni/e2e
+    ./get_tools.sh
+    CNI_VERSION="$CNI_VERSION" ./generate_yamls.sh
+    OCI_BIN="$OCI_BIN" ./setup_cluster.sh
```

Comment:
if one of those fails, popd wont happen and then the cleanup will fail because it is in the wrong directory
so you might want to keep the trap popd ?

Reply:
Right. I have run this locally, and checked that I got a clean repo after sending a SIGINT while running that section. 

Good catch; fixed it now, afaiu.

Reply:
btw on some machines / bash versions SIGINIT is needed to be added to the trap

nit: you might want to use 
`trap ... RETURN`
and then you don't need the `trap - EXIT` and the popd afterwards as it will do the popd in any case of function return (but verify please that it works on both good and bad path please if you try it)

Reply:
Done.

Reply:
I just hope that SIGINT wont trigger it twice for you now, as there is both SIGINT and RETURN now ")

Reply:
locally it didn't. And nobody will be sending SIGINTs in CI.